internal/karma/karma.conf.js

```diff
@@ -1,5 +1,59 @@
 // Karma configuration
 // GENERATED BY Bazel
+const path = require('path');
+const fs = require('fs');
+const tmp = require('tmp');
+
+process.env.CHROME_BIN = require('puppeteer').executablePath();
+
+let files = [
+  TMPL_files
+];
+
+const manifestFile = path.join(process.env.TEST_SRCDIR, "MANIFEST");
+if (fs.existsSync(manifestFile)) {
+  const manifest = {};
+  fs.readFileSync(manifestFile, { encoding: 'utf8' })
+    .split('\n')
+    .forEach((l) => {
+      const m = l.split(' ');
+      manifest[m[0]] = m[1];
+    });
+  const manifestKeys = Object.keys(manifest);
+  files = files.map((f) => {
+    if (manifestKeys.includes(f)) {
+      return manifest[f];
+    } else {
+      throw new Error(`File not found in MANIFEST: ${f}`);
+    }
+  });
+} else {
```

Comment:
do we need to support a case where there is no `MANIFEST`?

Reply:
same as comment below. `build_bazel_rules_typescript/external/` prefix is what is needed to match in the manifest and that needs to be dropped so it matches on runfiles.

Reply:
I can add a comment here that explains this

Reply:
Cleaned up. No MANIFEST case is not needed.